### PR TITLE
Home Page Styling 

### PR DESCRIPTION
**Pull request by daltonfury42**
Added a smoother font. Otherwise minor touchups to make the landing page look better:

image

### CODE REVIEW

**Comment by daltonfury42** [comment]
@maaverik I'll take care of smaller screens in sub-sequesnt PRs

File: simplq/src/components/pages/Home/LandingPage.jsx [hunk 1]
```diff
@@ -8,7 +8,11 @@ export default (props) => {
     <div className={styles['landing-page']}>
       <div>
         <Header className={styles['main-header']} text="SimplQ" />
-        <h3 className={styles.center}>A long overdue alternative to physical Queues</h3>
+        <p className={styles.subtitle}>A long overdue alternative to physical queues</p>
+        <p className={styles.description}>
+          Create and manage queues in instantly, free online queue management solution for easy
+          crowd control
+        </p>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Home/LandingPage.jsx:15]: We can replace with a better text once finalised

File: simplq/src/index.css [hunk 2]
```diff
@@ -4,8 +4,7 @@
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
-    'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  font-family: 'Quicksand', sans-serif;
```
Comment (daltonfury42) [inline simplq/src/index.css:7]: a more cleaner font I stole from https://artisanthemes.io/best-google-fonts-combinations-modern-agency-website/

File: simplq/src/styles/homePage.module.scss [hunk 3]
```diff
@@ -1,41 +1,54 @@
 @import 'utils';
 
-div {
-  @include div-padding();
-}
-
```
Comment (daltonfury42) [inline simplq/src/styles/homePage.module.scss:6]: We should not do this globally, would give unwanted side effects, so in our case, the text box from material ui gets unwanted padding becasue of this:

image

File: simplq/src/components/pages/Home/LandingPage.jsx (hunk 1 above)
Comment (maaverik) [reply simplq/src/components/pages/Home/LandingPage.jsx:15]: Maybe change it to 'Create and manage queues with our free, online queue management solution for easy and instant crowd control' for now.

Comment (daltonfury42) [reply simplq/src/components/pages/Home/LandingPage.jsx]: Done

**Review by maaverik — APPROVED** [review]

**Pull request merged**